x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -71,17 +73,47 @@ export const transactionGroupsOverviewRoute = createRoute({
       rangeRt,
       uiFiltersRt,
       t.type({
-        size: toNumberRt,
+        transactionType: t.string,
+        latencyAggregationType: latencyAggregationTypeRt,
+      }),
+    ]),
+  }),
+  options: {
+    tags: ['access:apm'],
+  },
+  handler: async ({ context, request }) => {
+    const setup = await setupRequest(context, request);
+
+    const searchAggregatedTransactions = await getSearchAggregatedTransactions(
+      setup
+    );
+
+    const {
+      path: { serviceName },
+      query: { latencyAggregationType, transactionType },
+    } = context.params;
+
+    return getServiceTransactionGroups({
+      setup,
+      serviceName,
+      searchAggregatedTransactions,
+      transactionType,
+      latencyAggregationType: latencyAggregationType as LatencyAggregationType,
+    });
+  },
+});
+
+export const transactionGroupsStatisticsRoute = createRoute({
```

Comment:
I don't think it makes sense to call this endpoint `/statistics`. The other API call also returns statistics. My suggestion was to call them both statistics (or stats), but differentiate between primary statistics and comparison statistics. (I don't think these are _great_ names so if we figure out something better that's fine too).

Reply:
We now have two APIs
- `GET /api/apm/services/{serviceName}/transactions/groups/overview`
Returns transactions grouped by name and latency, throughput, error rate and impact values

- `GET /api/apm/services/{serviceName}/transactions/groups/statistics`
Returns the latency, throughput, and error rate timeseries and impact.

We won't have a third one that will return the comparison data. When the comparison data is added, the second API is going to return the `currentPeriod` and `previousPeriod`, which contains the same data but in a different time range.

IMHO `/metrics` is what makes more sense here, but neither you nor @sqren liked it, so I changed to `/statistics` because it also makes sense to what the API returns. Besides these two options, I don't have a better name to give to this API 🤯.

Reply:
Metrics is an overloaded term (might be the processor event, might be system metrics, might be any kind of pre-aggregated data structure, might be the ES agg). They're both statistics. My suggestion earlier was to use `primary_statistics` instead of `overview` and `comparison_statistics` instead of `statistics`.

Reply:
don't think the comparison feature should drive the name of the APIs. If one day we decide to remove the comparison, we'd have to rename all APIs.

Reply:
If we change the intent of this API I think renaming is an acceptable and logical consequence. Feel free to suggest something else but I don't think overview vs statistics is a good distinction.

Reply:
well, since I don't have any other suggestion, I'll stick with yours 🤪